README.md

```diff
@@ -422,6 +422,10 @@ To learn how to get notifications when resource 1 changes, or how to use resourc
 ## Known issues
 
 1. Mutex issue using debug profile, issue #[303](https://github.com/ARMmbed/mbed-os-example-client/issues/303).
+1. UDP connection fails with on-line compiler (ARM CC 5.06 update 3) #[357](https://github.com/ARMmbed/mbed-os-example-client/issues/357).
+1. K64F LWIP IPv6 not working with older than ARM CC 5.06 update 6 #[359](https://github.com/ARMmbed/mbed-os-example-client/issues/359).
+
+ARM CC 5.06 update 3 has multiple compiler bugs, we recommend updating to [5.06 update 6](https://developer.arm.com/products/software-development-tools/compilers/arm-compiler/downloads/version-5).
 
```

Comment:
We need to be precise on the issue we're seeing, and therefore link to the issue (https://github.com/ARMmbed/mbed-os-example-client/issues/359)
I don't think we should ask to update to the latest compiler without a proper reasoning here.

Reply:
? I'm listing two issues right before that sentence.

Reply:
Sorry, I didn't understand what issue you are referring to. I assume it's the latter (LwIP), isn't?
Maybe correct the indentation and add "workaround" to make sure people understand the proposed solution?
Also please fix the name of the compiler: ARM CC --> Arm Compiler
Thanks!